src/components/Patient/PatientConsentRecords.tsx

```diff
@@ -107,6 +107,10 @@ export default function PatientConsentRecords(props: {
       crumbsReplacements={{
         [facilityId]: { name: patient?.facility_object?.name },
         [patientId]: { name: patient?.name },
+        consultation: {
+          name: "Consultation",
+          uri: `/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/update`,
```

Comment:
shouldn't this ideally be taken to the consultation dashboard instead of the consultation form?

```suggestion
          uri: `/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}`,
```